### ISSUE TITLE
Definition and axioms of 'scalar measurement datum'

### ISSUE THREAD

**Issue by APCox** [obi-ontology/obi#1263]
The current definition of 'scalar measurement datum' is: "a measurement datum that is composed of two parts, numerals and a unit label." This means that measurement datum items that utilize a scale without units (e.g. a Likert or other rating scale) must be typed as 'measurement datum' rather than 'scalar measurement datum'. Is this the intended usage? If so, then it would be useful to add a term such as 'ordinal measurement datum' as a subtype of 'measurement datum' (and possibly the parent of 'scalar measurement datum').

More importantly, there appears to be an unnecessary duplication of entities caused by the axiomatization of 'scalar measurement datum'. Specifically, the axioms for 'scalar measurement datum' include:
1. 'has measurement unit label' min 1 owl:Thing
2. 'has measurement value' min 1 rdfs:Literal
3. 'has value specification' some 'scalar value specification'

These axioms appear unproblematic until you notice that 'scalar value specification' includes the axioms:
1. 'has measurement unit label' min 1 owl:Thing
2. 'has specified numeric value' min 1 rdfs:Literal

such that the value and unit label are duplicated for both the scalar measurement datum and the scalar value specification. The result is 2 presumably distinct ICEs -- each of which has its own value and unit label. It is unclear to me what value is added by including axiom (3) given the current axioms. As far as I can tell, (3) currently creates an extra ICE with duplicate values. Clarification would be greatly appreciated.

**Comment by ddooley** [comment]
Thanks for raising this problem too - it needs resolution and OBI is committing to a solution before the end of January.  This duplication dates back to when OBI introduced value specifications as a second and improved way to capture the value associated with a datum (and separate scales of measure, so in a data structure allowing both a Celsius and Fahrenheit value on a temperature measurement datum); previous to that datums could have 'has measurement value' and 'has measurement unit label' data properties directly as you note.  

The unfinished business of that work was to drop these data properties from 'scalar measurement datum'.
I suggest we do that now.

An RDF data structure involving 'has unit' and 'has numeric value' properties (common in the non-OBI world), and OBI datum having 'has measurement unit label' and 'has measurement value' could be converted into a value specification unit and value for data compatibility and OWL driven reasoning.

**Comment by APCox** [comment]
Thank you for the response. Removing the duplication of axioms is good, but I am wondering whether the definition (and relevant axiom) of 'scalar measurement datum' will also be modified such that it does not require that instances of it have a measurement unit label? Doing so would allow data that use unitless scales to be represented as instances of scalar measurement datum.

Either way, this particular decision affects a large number of terms in my current ontology effort.

**Comment by ddooley** [comment]
Right, the 'scalar measurement datum' is still carrying requirement for measurement unit label.  A number of OBI curators don't want to see a loosening of that, but I can double check next monday.  If it isn't loosened then it calls for a "unitless scalar measurement datum" class I think.  As you say this is required for survey datum Likert scales etc.  I take it you work with alot of survey data?

**Comment by jamesaoverton** [comment]
Discussed again on 2021-01-25: We'll go ahead and drop these axioms from 'scalar measurement datum':

1. 'has measurement unit label' min 1 owl:Thing
2. 'has measurement value' min 1 rdfs:Literal

Since this could potentially break things for people, we'll give some warning. The process will be:

1. @ddooley will make a PR with these changes
2. @jamesaoverton will announce to obi-devel and obi-users, with a 2 week comment period
3. Unless there are major objections, @jamesaoverton will merge

**Comment by ddooley** [comment]
'has measurement unit label' and 'has measurement value' have been removed in:
https://github.com/obi-ontology/obi/pull/1299

**Comment by ddooley** [comment]
This is being addressed in https://github.com/information-artifact-ontology/IAO/pull/241 which requires feedback from OBI/IAO users.

**Comment by ddooley** [comment]
Closing this now too as new data model will handle.
